lib/ansible/modules/cloud/azure/azure_rm_batchaccount_info.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+#
+# Copyright (C) 2019 Junyi Yi (@JunyiYi)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_batchaccount_info
+version_added: "2.9"
+short_description: Gather info for Azure Batch Account
+description:
+    - Gather info for Azure Batch Account.
+
+options:
+    resource_group:
+        description:
+        - The name of the resource group in which to create the Batch Account.
```

Comment:
indention

Reply:
Is it possible to use this type of indentation since this indentation is also valid in yaml? Actually I do not have control over this indentation, it is restricted by the Ruby library and Google's code also uses this type of indentation.

Reply:
This is not consistent with current Ansible indention style. let's see community feedback.